test/actor.test.ts

```diff
@@ -67,7 +67,7 @@ describe('spawning machines', () => {
       },
       SET_COMPLETE: {
         actions: send('SET_COMPLETE', {
-          to: (ctx, e) => {
+          to: (ctx, e: Extract<TodoEvent, { type: 'SET_COMPLETE' }>) => {
```

Comment:
`send` action doesn't seem to have event inferred correctly, probably something which we could improve in the future

Reply:
I agree, let's make an issue to track it.